mff_rams_plugin/templates/kiosk_printing/print_badges.html

```diff
@@ -2,7 +2,7 @@
 <head>
     <!-- Print and Reload Function -->
     <script type="text/javascript">
-        window.print();
+        //window.print();
```

Comment:
Just delete this line instead of leaving it hanging around.

Reply:
Removed